### ISSUE TITLE
Unusable window resolution in ChromeOs

### ISSUE THREAD

**Issue by Gallowglass77** [syl20bnr/spacemacs#14076]
I just installed Spacemacs (twice to be sure) to the Linux Beta on CromeOS. The splash screen and edit screen is in tiny resolution and is unusable.  <ctrl>-+ has no effect. <ctrl>-+ works in Doom emacs and in other terminal windows. I think this is an issue because a new install should be usable without adjusting fonts / resolution immediately. Note I added Nanum-Gothic and Source_Code_Pro ttf files to /usr/local/share/fonts before the install.

![Tiny_Spacemacs](https://user-images.githubusercontent.com/72059114/96754114-b6008e80-1385-11eb-9ed3-04663bb7029a.png)

**Comment by duianto** [comment]
I don't have a HiDPi display to test with.

But this should really be handled automatically.
I'm sure Emacs can check the current resolution.

Hopefully someone with a HiDPi display can help.

---

The text resizing is also an issue.

The default Emacs text scale key bindings are:
larger text:
- `C-x C-+` 
- `C-x C-=`

smaller text:
- `C-x C--`

reset size:
- `C-x C-0`

After pressing one of them once, then the minibuffer shows:
`Use +,-,0 for further adjustment`

And now it's enough to just press:
`=`, `+`, `-` or `0`
without the `C-` (ctrl) prefix.

However since the minibuffer is focused when answering the setup questions,
this means that only the text in the minibuffer changes size.

**Comment by Gallowglass77** [comment]
The Emacs text size bindings work as indicated above and only change the mini buffer, Also the small size reverts once you answer the configuration question. Final point is configuration seems to have had a number of errors. Looks like Chromebook Linux is not ready for Spacemacs,
![Screenshot 2020-10-25 at 8 23 57 PM](https://user-images.githubusercontent.com/72059114/97131318-13d40400-1701-11eb-849e-88e0c1a6616b.png)

**Comment by duianto** [comment]
Your first screenshot shows that your on the Spacemacs `master` branch version: `0.200.13`
With Emacs 26.1.

A gpg key issue was fixed in 26.3.

But it would probably be better if you were able to install the latest stable version: Emacs 27.1

The Spacemacs `develop` branch `0.300.0`, is the recommended branch until the next version is released 
(I have no insight into when that is).

#### To try the `develop` branch:
Backup your current setup and make sure that there isn't a `.emacs.d` directory or a `.spacemacs` file in your home directory. Then navigate to your home directory in a terminal (or command prompt) and clone directly to the `develop` branch with:
```
git clone -b develop https://github.com/syl20bnr/spacemacs .emacs.d
```

If your able to install Emacs 27.1 and switch to the Spacemacs `develop` branch, then the old Emacs version and Spacemacs branch could be eliminated as the source of the issues.

**Comment by Gallowglass77** [comment]
That switch to the develop branch seemed to fix the issue. Spacemacs loaded in a legible mode and I was able to use the emacs <c>-+ bindings to further change resolution when editing. Thank you :-). Note i have to run and quite twice to get all packages to to install.
